dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/system/SystemUpdateService.java

```diff
@@ -50,9 +50,9 @@
 import org.hisp.dhis.user.UserCredentials;
 import org.hisp.dhis.user.UserCredentialsStore;
 import org.hisp.dhis.user.UserService;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
 
-import com.google.api.client.http.HttpStatusCodes;
```

Comment:
should this one also be banned?

Reply:
As spring has something like it I think so yes. If you cannot use the spring one this is a good indication that the code is in the wrong module doing HTTP stuff in a non web facing module.

Reply:
We should definitely do something about it but should we ban it or exclude it?

I think it make sense to ban the JUnit4 dependency because we cannot really remove it, but it this case (and also the wildfly-common) maybe we can just remove it. In both cases they just come from one dependency in the pom.
What do you think? @teleivo @jbee

Reply:
So my guess is the package comes from dependency https://developers.google.com/api-client-library/java
If we are interacting with google, removing this client library is probably not a good or convenient option. We might still be able to ban these imports. @jbee could there be a module in this repo where using these codes would be ok? If so we shouldn't ban these imports in every module.

Reply:
This would need some investigation into how far we use the library and what for. I am not aware of any usage so I'd hope its just a few smaller cases.

Reply:
My suggestion was to just remove `com.google.http-client:google-http-client-apache-v2` that is part of the `api-client-library`. We are not using it anywhere.
In the end banning or excluding a specific library has the same effect, the only difference is that banning will appear when compiling with maven while if we exclude the (sub)dependency it would not be possible to add it while coding

Reply:
that sounds good to me then 😄